tests/foreman/ui/test_user.py

```diff
@@ -23,7 +23,7 @@
 from robottelo.ui.session import Session
 
 
-def valid_strings(len1=100):
+def valid_strings(len1=10):
```

Comment:
When we use long string length e.g. `test1223455`, UI truncates the string and ends it with `test122....` so test failed at asserting the original string. This is a common issue for other UI entities/tests too. So I changed it to default '10'. We have opened a github issue #1796 to address this.